axelrod/strategies/_strategies.py

```diff
@@ -197,4 +197,5 @@
     ZDGen2,
```

Comment:
Unless I'm missing it, you need to add `AdaptiveTitForTat` to the `all_strategies` list. I believe that might trigger a failure in the docstrings but that should just require incrementing a count :)